WiseBuy/View Controllers/RecommendationFeedViewController.m

```diff
@@ -54,10 +67,12 @@ -(void) queryPosts {
     [query includeKey:kPostedBy];
     [query whereKey:kPostedBy containedIn:user[kFriendList]];
     
-    [query findObjectsInBackgroundWithBlock:^(NSArray * _Nullable posts, NSError * _Nullable error) {
+    [query findObjectsInBackgroundWithBlock:^(NSArray<Post *> * _Nullable posts, NSError * _Nullable error) {
         if (posts) {
             self.posts = posts;
             [self.tableView reloadData];
+            [progressHUD dismissAfterDelay:0.1 animated:YES];
```

Comment:
What's the delay for?

Reply:
thank you. i should have fixed this

Reply:
initially it was only for me to easily see if the loading state worked